packages/eslint-config/eslint-builtin-rules.js

```diff
@@ -1,10 +1,14 @@
-{
+/* !
```

Comment:
Made these JavaScript files instead of JSON because importing JSON it's still an experimental JS feature.